akka-actor-tests/src/test/scala/akka/routing/SmallestMailboxSpec.scala

```diff
@@ -19,11 +19,11 @@ class SmallestMailboxSpec extends AkkaSpec with DefaultTimeout with ImplicitSend
       val usedActors = new ConcurrentHashMap[Int, String]()
       val router = system.actorOf(SmallestMailboxPool(3).props(routeeProps = Props(new Actor {
         def receive = {
-          case (busy: TestLatch, receivedLatch: TestLatch) =>
+          case (busyLatch: TestLatch, receivedLatch: TestLatch) =>
```

Comment:
Sounds almost annoying, to not be able to re-use the var name. The good thin could be to not access outside context, accidentally.

Reply:
I think there was two actual tests doing the wrong thing I found from it though